.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -669,6 +669,32 @@ popd > /dev/null
 chown -R ${DAEMON_USER}:${COURSE_BUILDERS_GROUP} ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 chmod -R 555 ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 
+################################################################################################################
+################################################################################################################
+# BUILD AND INSTALL ANALYSIS TOOLS TS
+
+echo -e "Build and install analysis tools ts"
+
+mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS
+
+ANALYSIS_TOOLS_TS_REPO=${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/AnalysisToolsTS/
+
+# Copy cloned files to AnalysisToolsTS directory
+rsync -rtz ${ANALYSIS_TOOLS_TS_REPO} ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS
+
+# pushd ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS
+
+# # install npm packages
+# npm install
+
+# # build project
+# npm run build
```

Comment:
Is there any particular reason why we don't want to go ahead and build the project here, even if we aren't using it for anything yet?  This isn't necessarily something that I'm requesting be done in this PR, just a general question.

If the installation scripts aren't run, I think it would be good to at least put some sort of placeholder message here to indicate that installation was skipped.

Reply:
Uncommented. Tested the procedure by running `submitty_install`